### PR TITLE
app-arch/par2cmdline-turbo: new package, add 1.1.0, 9999

### PR DESCRIPTION
**Pull request by joecool1029**
I want to add this package to Gentoo and proxy maintain it as I already use it with net-nntp/sabnzbd and it should be drop-in compatible for most arches and packages that use par2cmdline. 

This is the SIMD optimized version and way faster than app-arch/par2cmdline. Please let me know if there's anything else needed to get it included. 

EDIT: fixed the QA issues.

### CODE REVIEW

File: app-arch/par2cmdline-turbo/par2cmdline-turbo-1.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+EAPI=8
+
+inherit autotools toolchain-funcs
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/animetosho/par2cmdline-turbo"
+else
+	SRC_URI="https://github.com/animetosho/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86 ~amd64-linux"
```
Comment (thesamesam) [inline app-arch/par2cmdline-turbo/par2cmdline-turbo-1.0.1.ebuild:13]: Let's drop everything but ~amd64 please pending rekeywording. I assume these are inherited from par2cmdline but I wouldn't be surprised if the SIMD stuff breaks it.

File: app-arch/par2cmdline-turbo/par2cmdline-turbo-1.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_test() {
+	# test22 fails when run in parallel
```
Comment (thesamesam) [inline app-arch/par2cmdline-turbo/par2cmdline-turbo-1.0.1.ebuild:41]: Reported that bit upstream?

Comment (joecool1029) [reply app-arch/par2cmdline-turbo/par2cmdline-turbo-1.0.1.ebuild:41]: This was inherited from app-arch/par2cmdline (upstream of this project). Maintainer of the par2cmdline-turbo project isn't interested in bugs inherited from them (their policy stated on frontpage of project), and upstream does not appear to be active.

**Comment by thesamesam** [comment]
Merged (squashed), thanks!

**Pull request closed without merging**